### PR TITLE
#2092 Fix skill tester rating predictions

### PR DESCRIPTION
**Pull request by wsbrenk**
1. changes proposed in this pull request:

   - fixes issue #2092
   - if not fixing an existing issue, comments explaining the purpose of the PR should be provided)

2. `src/main/resources/release_notes.md` ...

 - [x] has been updated
 - [ ] does not require update

3. [Optional] suggested person to review this PR @tychobrailleur

### CODE REVIEW

**Comment by wsbrenk** [comment]
@tychobrailleur what is your opinion: Should we change version to 8.1? Would be a little work on release_notes and changelog.

File: src/main/java/module/playerOverview/SpielerTrainingsSimulatorPanel.java [hunk 1]
```diff
@@ -187,30 +192,51 @@ public void setSpieler(Player player) {
         repaint();
     }
 
+    /**
+     * Initialize temporal player instance with combo box selections
```
Comment (tychobrailleur) [inline src/main/java/module/playerOverview/SpielerTrainingsSimulatorPanel.java:196]: “temporal” : temporary?

Comment (wsbrenk) [reply src/main/java/module/playerOverview/SpielerTrainingsSimulatorPanel.java]: thanks

**Comment by wsbrenk** [comment]
merge request #2094 will change ho version to 8.1 and initializes changelog and release_notes.
should be merged before this one.

**Pull request merged**